From: Moni Shoua <monis@mellanox.com>
Date: Thu, 14 Jan 2016 17:48:07 +0200
Subject: net/mlx4: Remove unused macro
Patch-mainline: v4.5-rc1
Git-commit: f25bf1977f7a968e85fe8ab99252b8132c6cf8c4
References: bsc#966191 FATE#320230 bsc#966186 FATE#320228

The macro mlx4_foreach_non_ib_transport_port() is not used anywhere. Remove it.

Fixes: aa9a2d51a3e7 ("mlx4: Activate RoCE/SRIOV")
Signed-off-by: Moni Shoua <monis@mellanox.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 include/linux/mlx4/device.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/include/linux/mlx4/device.h
+++ b/include/linux/mlx4/device.h
@@ -978,10 +978,6 @@ struct mlx4_mad_ifc {
 	for ((port) = 1; (port) <= (dev)->caps.num_ports; (port)++)	\
 		if ((type) == (dev)->caps.port_mask[(port)])
 
-#define mlx4_foreach_non_ib_transport_port(port, dev)                     \
-	for ((port) = 1; (port) <= (dev)->caps.num_ports; (port)++)	  \
-		if (((dev)->caps.port_mask[port] != MLX4_PORT_TYPE_IB))
-
 #define mlx4_foreach_ib_transport_port(port, dev)                         \
 	for ((port) = 1; (port) <= (dev)->caps.num_ports; (port)++)	  \
 		if (((dev)->caps.port_mask[port] == MLX4_PORT_TYPE_IB) || \
